docs/lpd_data.ipynb

```diff
@@ -287,11 +287,11 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "source": [
+    "LPD data may also be recorded in *parallel gain* mode, resulting in high-, medium- and low-gain frames for each pulse. To read this kind of data with the correct labels, use `LPD1M(run, parallel_gain=True)`."
```

Comment:
"... resulting in high-, medium- and low-gain, presented as 0, 1, and 2, respectively, frames ..."